Templates/AppSource App/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -1,6 +1,6 @@
-name: '_Build AL-GO project'
+name: '_Build AL-Go project'
 
-run-name: 'Build project ${{ inputs.project }}'
+run-name: 'Build AL ${{ inputs.project }}'
```

Comment:
Personally, I prefer Build project 🤷‍♂️ But I'm okay with changing it if you feel strongly about it. Could also just be "Build ${{ inputs.project }}?

Reply:
Build is better I think